opentreemap/treemap/ecobenefits.py

```diff
@@ -80,6 +80,21 @@ def benefits_for_object(self, instance, obj):
         return {}
 
 
+class CountOnlyBenefitCalculator(BenefitCalculator):
+    def __init__(self, clz):
+        self.clz = clz
+
+    def benefits_for_filter(self, instance, item_filter):
+        features = item_filter.get_objects(self.clz)
+        return ({},
+                {'resource':
+                 {'n_objects_used': 0,
+                  'n_objects_discarded': len(features)}})
```

Comment:
[`get_objects`](https://github.com/OpenTreeMap/otm-core/blob/4f6c657f2337d766a10268d9d422de7e082c22e8/opentreemap/treemap/search.py#L54-L84) returns a queryset, so I think you can change `len(features)` here to be `features.count()`.